concepts/05 UI Components/TagBox/00 Getting Started with TagBox/60 Configure Multiple Tags Selection.md

```diff
@@ -0,0 +1,71 @@
+Use the following properties to adapt the TagBox text field to multiple selected tags:        
+
+- [multiline](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#multiline)     
+Specifies whether to display tags on multiple lines when the tags overflow the component's width.
```

Comment:
Alright. This is a part of a getting started topic, am I right? If yes, you should write in imperative mood. What a user has to do to "adapt" the tag box as you say? Set the multiline property to true to allow the tag box to display tags on multiple lines?

Reply:
Imperative is at the beginning of the list, isn't it enough? We tell users which properties to use and then what these properties do.

Reply:
OK. Then you need to decide what this section is? Is it a proper set of instructions or an optional overview section in your tutorial?

Reply:
Sounds reasonable